desktop/.gitignore

```diff
@@ -0,0 +1,2 @@
+/bin
+/bin
```

Comment:
I'd make sure /bin is listed once, and I'd make sure it is being ignored properly, but yes: delete the duplicates.